src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4106,7 +4106,167 @@
+    <value>The variable name is invalid.</value>
+    <comment>Error that occurs during graping when a variable name is invalid.</comment>
+  </data>
+  <data name="BracketMismatch" xml:space="preserve">
+    <value>The equation is missing an opening bracket.</value>
+    <comment>Error that occurs during graping when a { is missing</comment>
+  </data>
+  <data name="UnmatchedBracket" xml:space="preserve">
+    <value>The equation is missing a closing bracket.</value>
+    <comment>Error that occurs during graping when a } is missing.</comment>
+  </data>
+  <data name="CannotUseIInReal" xml:space="preserve">
+    <value>"i" and "I" cannot be used as variable names.</value>
+    <comment>Error that occurs during graping when i or I is used.</comment>
+  </data>
+  <data name="GeneralError" xml:space="preserve">
+    <value>The equation could not be graphed.</value>
+    <comment>General error that occurs during graping.</comment>
+  </data>
+  <data name="InvalidNumberDigit" xml:space="preserve">
+    <value>The digit could not be resolved for the given base.</value>
+    <comment>Error that occurs during graping when trying to use bases incorrect. Ex: base(2,1020).</comment>
+  </data>
+  <data name="InvalidNumberBase" xml:space="preserve">
+    <value>The base must be greater than 2 or less than 36.</value>
+    <comment>Error that occurs during graping when the base is out of range.</comment>
+  </data>
+  <data name="InvalidVariableSpecification" xml:space="preserve">
+    <value>A function in the equation requres a variable in a certain argument position.</value>
+    <comment>Error that occurs during graping when a function requires a variable in a particular position. Ex: 2nd argument of deriv.</comment>
+  </data>
+  <data name="ExpectingLogicalOperands" xml:space="preserve">
+    <value>Equation is mixing logical and scalar operands.</value>
+    <comment>Error that occurs during graping when operands are mixed. Such as true and 1.</comment>
+  </data>
+  <data name="CannotUseIndexVarInOpLimits" xml:space="preserve">
+    <value>x or y cannot be used in the upper or lower limits.</value>
+    <comment>Error that occurs during graping when x or y is used in integral upper limits.</comment>
+  </data>
+  <data name="CannotUseIndexVarInLimPoint" xml:space="preserve">
+    <value>x or y variable cannot be used in the limit point.</value>
+    <comment>Error that occurs during graping when x or y is used in the limit point.</comment>
+  </data>
+  <data name="CannotUseComplexInfinityInReal" xml:space="preserve">
+    <value>Cannot use complex infinity.</value>
+    <comment>Error that occurs during graping when complex infinity is used</comment>
+  </data>
+  <data name="CannotUseIInInequalitySolving" xml:space="preserve">
+    <value>Cannot use complex numbers in inequalities.</value>
+    <comment>Error that occurs during graping when complex numbers are used in inequalities.</comment>
```

Comment:
I did not notice any errors that were longer than one sentence. Can we remove all periods?